version_control/katalyst_by_codurance/css/templates/home.css

```diff
@@ -88,26 +88,23 @@
 
 /* Methodology Section */
 
-.methodology {
-    background: var(--dark-blue);
-    padding: var(--space-3xl) 0;
+.methodology__content {
+        background: var(--dark-blue);
+        padding: var(--space-2xl) var(--space-m);
+        color: white;
 }
 
 .methodology .section-header {
     text-align: left;
-}
-
-.methodology h2,
-.methodology p {
-    color: white;
+    margin-inline: inherit;
+    max-width: 65ch;
```

Comment:
On the design the width on this content now seems to be a little bit lower, about 52ch, so maybe we can try a value similar to that

Reply:
I rather talk to Ursula and have a defined max character count for block of texts, because we would end up setting random max-widths here and there.